### PR TITLE
Task #11479: Administrators manager users

### PR DESCRIPTION
**Pull request by huynhlv**
Screenshot from 2019-04-22 19-41-27
LinkRedmi: https://edu-redmine.sun-asterisk.vn/issues/11479

### CODE REVIEW

**Comment by huynhlv** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Admin::UsersController < ApplicationController
+  before_action :admin_user
+  before_action :load_user, except: :index
```
Comment (VuThiTranVan) [inline app/controllers/admin/users_controller.rb]: only destroy nhé
vì có 2 action thôi mà :D

**Comment by huynhlv** [comment]
ready

**Comment by VuThiTranVan** [comment]
commented
fix conflict code

**Comment by huynhlv** [comment]
ready

**Pull request closed without merging**